bindings/python/examples/exchange/4_listen_events.py

```diff
@@ -35,8 +35,8 @@ def callback(event):
     received_event = True
 
 
-# Only interested in new outputs here.
-wallet.listen(callback, [WalletEventType.NewOutput])
+# Only interested when inputs are being selected.
+wallet.listen(callback, [WalletEventType.SelectingInputs])
```

Comment:
Why did you change this? Input selection will not run in this example and makes no sense for it anyways :sweat_smile:

Reply:
Right, I went too fast replacing, let me see

Reply:
Yeah, this is wrong.

Reply:
Ah sorry I linked the wrong event file, the other is added in another PR and there is https://github.com/iotaledger/iota-sdk/issues/2121 so no point in you doing any changes now

Reply:
Okay, just reverted those changes which definitely did not make any sense, but, now I fail to see what exactly is there to change in the python bindings? `WalletEventType` looks just fine

Reply:
Yeah nothing to change in this PR, the TransactionProgress part is just completely missing, but will be added in another PR

Reply:
yeah, saw that too, but since it's the same for the other variants I supposed it was normal